go.mod

```diff
@@ -198,9 +198,7 @@ require (
 )
 
 replace (
-	// SDK with blocktime in GRPC query: https://github.com/celestiaorg/cosmos-sdk/pull/337
-	github.com/cosmos/cosmos-sdk => github.com/celestiaorg/cosmos-sdk v1.15.0-sdk-v0.46.13.0.20230731181258-40cbd617f1ba
-	// github.com/cosmos/cosmos-sdk => github.com/celestiaorg/cosmos-sdk v1.15.0-sdk-v0.46.13
+	github.com/cosmos/cosmos-sdk => github.com/celestiaorg/cosmos-sdk v1.15.0-sdk-v0.46.13
```

Comment:
[question] is this blocked on a release with https://github.com/celestiaorg/cosmos-sdk/pull/337 ?

Reply:
now not any more